app/assets/javascripts/incidents/batch_export.coffee

```diff
@@ -0,0 +1,35 @@
+alreadyExportedWarning = (count) ->
+  words = if count == 1
+    ['incident has', 'this incident']
+  else ['incidents have', 'these incidents']
+  """
+  #{count} selected #{words[0]} already been exported.
+  Please confirm that you would like to re-export #{words[1]}.
+  """
```

Comment:
<img width="512" alt="screen shot 2018-01-30 at 6 54 27 pm" src="https://user-images.githubusercontent.com/3988134/35598124-7e4d1a04-05ef-11e8-8603-6f55447e8786.png">
<img width="512" alt="screen shot 2018-01-30 at 6 54 36 pm" src="https://user-images.githubusercontent.com/3988134/35598125-7e62cd36-05ef-11e8-88ee-380662584d14.png">

Reply:
I changed 'doublecheck' to 'confirm' after taking these screenshots.